tools/lc

```diff
@@ -44,6 +44,7 @@ cmd_project_new() {
 
   echo "—> generating IDL entry point"
   cp ./tools/templates/idl.djinni $project_name/idl/
+  cp ./tools/templates/callback.djinni $project_name/idl/
```

Comment:
Ok I think I got it ! We have callback template we inject into each coin because djinni is f***ked up with its `extern` | `import` ?

Reply:
Yep… :(